### PR TITLE
Misc. typos and whitespace fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: src/Mod/Fem/feminout/importZ88Mesh.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def read_z88_mesh(z88_mesh_input):
                     input_continues = False
                 elif z88_element_type == 16:
                     # volume16 Z88 --> tetra10 FreeCAD
-                    # N1, N2, N4, N3, N5, N8, N10, N7, N6, N9, , Z88 to FC is differend as FC to Z88
+                    # N1, N2, N4, N3, N5, N8, N10, N7, N6, N9, , Z88 to FC is different as FC to Z88
```
Comment (luzpaz) [inline src/Mod/Fem/feminout/importZ88Mesh.py]: @berndhahnebach is this correct ?

File: src/Mod/Fem/feminout/importZ88Mesh.py [hunk 2]
```diff
@@ -373,7 +373,7 @@ def write_z88_mesh_to_file(femnodes_mesh, femelement_table, z88_element_type, f)
                     n[3], n[1], n[2], n[0]))
         elif z88_element_type == 16:
             # tetra10 FreeCAD --> volume16 Z88
-            # N1, N2, N4, N3, N5, N9, N8, N6, N10, N7, FC to Z88 is differend as Z88 to FC
+            # N1, N2, N4, N3, N5, N9, N8, N6, N10, N7, FC to Z88 is different as Z88 to FC
```
Comment (luzpaz) [inline src/Mod/Fem/feminout/importZ88Mesh.py]: @berndhahnebach as well as this ?

File: src/Mod/Sketcher/Gui/CommandConstraints.cpp [hunk 3]
```diff
@@ -966,7 +966,7 @@ class DrawSketchHandlerGenConstraint: public DrawSketchHandler
         Gui::Selection().rmvSelectionGate();
         Gui::Selection().addSelectionGate(selFilterGate);
 
-        // Constrait icon size in px
+        // Constrain icon size in px
```
Comment (luzpaz) [inline src/Mod/Sketcher/Gui/CommandConstraints.cpp]: is this correct ?

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**